include/cppoptlib/problem.h

```diff
@@ -22,6 +22,7 @@ class Problem {
  public:
   Problem() {}
   virtual ~Problem()= default;
+  virtual bool check_x(const TVector &x){return true;}
```

Comment:
@spinicist if you remove your comment add click on "Review changes" selecting "Request changes" your comments will be marked as pending.

> This should be added to BoundedProblem, not Problem.